packages/ai/tests/type-check.test.ts

```
@@ -1,13 +1,26 @@
 /**
- * Type-level tests for TextActivityOptions
+ * Type-level tests for TextActivityOptions and TypedStreamChunk
  * These should fail to compile if the types are incorrect
  */
 
 import { describe, it, expectTypeOf } from 'vitest'
```

ESLint

[error] Member 'expectTypeOf' of the import declaration should be sorted alphabetically.

(sort-imports)